guides/Web_Frameworks/Pheonix.md

```diff
@@ -0,0 +1,28 @@
+# Pheonix
+[Pheonix](http://phoenixframework.org/) is a framework written in Elixir which implements the server-side MVC pattern. It is built for web development.It is a good option today in the market.The website of the pheonix says **A productive web framework that does not compromise speed and maintainability**.This is true in fact because earlier developers had a myth either speed or quality both can not go hand in hand one need to compromise the either but pheonix breaks the myth and therefore it carved its name among the most popular and fastest web frameworks.Phoenix is a modern web framework that helps building APIs and web applications easy. It’s built with [Elixir](https://elixir-lang.org/) and runs on Erlang VM .It has excellent support for handling very large numbers of simultaneous users.
+
+# Installaion
+The link for the installation is [here](https://hexdocs.pm/phoenix/installation.html#content).Pheonix has been provided with rich set of guides to understand the workflow.The link to the [documentation](https://hexdocs.pm/phoenix/Phoenix.html) is here.If you fallen in love with it at the very first sight you can also contribute to it [here](https://github.com/phoenixframework/phoenix).
+
+# What about learning Phoenix?
+There are dozens of resources available on the web.The best is if one can scratch the surface with some good tutorials isn't? There are few resources available one is [head start guide](www.phoenixforrailsdevelopers.com) and this is also a [good book](https://pragprog.com/book/phoenix/programming-phoenix).
+
+# Why Choosing Pheonix ?
```

Comment:
I don't want basic advantages. I want the lucid explanation of how Phoenix works. :smile: